### ISSUE TITLE
Sprint 28 Product support team priorities

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#4682]
🥅   — Sprint goals (pending Sprint planning)
👥   — Needs discussion, meetings and/or refining 

## Facilities 

[Agenda for weekly sync with Facilities team](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/product-team-support/product-team-weekly-syncs/facilities-cms-weekly-sync.md)

#### #4374 Local editor experience

- [x] #4481 Spike: Dashboard architecture cont'd  👥 
- [x] #4422 Implement form design 👥  
- [x] #4531 Implement TOCs and Fieldgroups for Vet Center form 
- [ ] #4477 Spike: modals for entity reference forms (forms within a form).
- [ ] Stretch: Implement dialog approach from #4477 if possible. 

#### #4370 Migration @kaise-lafrai, @swirtSJW

- [x] #4325 Update VAST migration for Vet Center nodes (WIP, needs testing) @kaise-lafrai 🥅
- [ ] #4369 Spike: How to handle changes to facility names in VAST?  @swirtSJW (stretch)
- [ ] #4410 Lighthouse discovery: Vet Center CAP and Outstation data @swirtSJW 
- [x] #4558 Update push of Vet Center operating status to include Outstations
- [x] #4445 Plan cutover for MVC and OS Vet Center Facility Status @kevwalsh 
- [x] #4564 Change management for Outstation facility status (national editors) (timing unclear ATM)
- [ ] #4411 Push Vet Center CAP data to Lighthouse (29)

#### #4318 Content model and governance

- [x] #4685 Set default value for Vet Center page introduction
- [x] #4408 Document Vet Center content model for Facilities team @nataliedunbarCS @kevwalsh  🥅 
- [x] #4615 - Refactor centralized content fields on Vet Center to use entity IDs and put _cc_ in their machine name
- [x] #4613 - Add `field_cc_vet_center_faqs` field to Vet Center content type
- [x] #4655 field_office on Vet Center Locations List should only allow vet_center content type, not MVC, OS, or CAP 
- [ ] #4541 Extend field_office entity reference permissions to Vet Center content types (dependent on #4325 migration)

Stretch:
- [ ] #4582 Hide children of "Vet Centers" section in the toolbar menu and add groupings (dependent on grouping)
- [ ] #4628 Make revision log mandatory for all vet_center_* content types

#### #4373 National editor experience

- [x] #4415 Add Vet Center specific fields to /vha-health-services view 

### VAMC System

- [x] #4441 Launch 8-10 VAMCs in VISN 6 and beyond @swirtSJW  @kevwalsh

Training
- [x] #3278 Training for Policies page
- [ ] #3277 User guide for Policies Page
- [ ] #4669 Update VAMC training videos

#### Governance & editorial experience (all stretch goals)

**Epic Centralized content national editing experience**  
- [ ] #4195 Support tokens in EFF for inserting local values into nationally managed content 
- [ ] #4056 Prevent deletion of centralized content  *needs refining* 
- [ ] #4016 Modify cc_documentor paragraph to have special form view  @swirtSJW *needs refining*

**Epic: Enforce IA design for VAMC editors**
- [ ] #4311 SPIKE: technical implementation of VAMC system menu IA governance @ethanteague 

**Epic: Unique content validation**
- [ ] #4074 Spike: Unique content validation approach   (will span more than one sprint) @kaise-lafrai 
  

### Campaign Landing Page MVP

[Agenda for weekly sync with Public Websites team](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/product-team-support/product-team-weekly-syncs/public-websites-cms-weekly-sync.md)

**#2275 Change management for beta users**  👥

- [x] #4472 Facilitate CLP beta test with VBA @nataliedunbarCS @xiongjaneg 👥  (needs a differnet issue id)

### Resources and Support

N/A

### VA Forms 

N/A

## Core team collaboration

**Drupal 9 update**
- [ ] #4283 Update Migration Tools module @swirtSJW 

**Core UX debt (from VAMC product)**
- [ ] #3292  Remove entity embed (not critical, could be moved to Sprint 27) 
  - [ ] #3293  Audit and fix detail pages that have used Entity Embed within rich text fields @kevwalsh 
  - [ ] #3294  Remove Entity embed tool from Rich text editor
